sentry-android-core/src/main/java/io/sentry/android/core/AndroidLogger.java

```diff
@@ -39,39 +36,26 @@ public void log(SentryLevel level, String message, Throwable throwable) {
       case FATAL:
         Log.wtf(tag, message, throwable);
         break;
-    }
-  }
-
-  SentryLevel toSentryLevel(int logcatLevel) {
-    switch (logcatLevel) {
-      case Log.VERBOSE:
-      case Log.DEBUG:
-        return SentryLevel.DEBUG;
-      case Log.INFO:
-        return SentryLevel.INFO;
-      case Log.WARN:
-        return SentryLevel.WARNING;
-      case Log.ERROR:
-        return SentryLevel.ERROR;
-      case Log.ASSERT:
+      case LOG:
+      case DEBUG:
       default:
-        return SentryLevel.FATAL;
+        Log.d(tag, message, throwable);
+        break;
     }
   }
 
   private int toLogcatLevel(SentryLevel sentryLevel) {
     switch (sentryLevel) {
-      case DEBUG:
-        return Log.DEBUG;
       case INFO:
         return Log.INFO;
       case WARNING:
         return Log.WARN;
       case FATAL:
         return Log.ASSERT;
-      case ERROR:
+      case LOG:
+      case DEBUG:
       default:
```

Comment:
Sorry just spotted this and afaik default is error for issue events. 
But we should not be adding level by default so the server infers that

Reply:
this is only used for calling the correct logging, not related to sentryEvent.level nor breadcrumb.level

Reply:
My bad.